src/sentry_plugins/slack/plugin.py

```diff
@@ -152,18 +166,18 @@ def notify(self, notification):
 
         # They can be the same if there is no culprit
         # So we set culprit to an empty string instead of duplicating the text
-        if culprit and title != culprit:
+        if self.get_option('include_culprit', project) and culprit and title != culprit:
```

Comment:
Got it - thanks for the pointers! I've updated the PR to s/exclude/include/, and added a couple test cases.